index.html

```diff
@@ -13,6 +13,8 @@
 		<meta property="og:url" content="https://gra0007.github.io/maimai-song-list/">
 		<meta name="twitter:card" content="summary_large_image">
 
+		<script src="https://unpkg.com/wanakana"></script>
```

Comment:
Can you move this to the body before the other script? That way the page can load first.

Reply:
oh whoops totally didn't see that other script there, hhehehhhHEHAHhehehHEH

Alright done